yarn.lock

```diff
@@ -23,6 +23,18 @@
     mrz "^4.2.0"
     tsyringe "^4.8.0"
 
+"@2060.io/credo-ts-didcomm-mrtd@^0.0.9":
```

Comment:
It looks good, the only thing missing is the update of MRTD v0.0.9 in the service agent models, so that the yarn lock doesn't manage two versions of the same library.